tests/acceptance/bootstrap/CollaborationContext.php

```diff
@@ -164,6 +162,35 @@ public function thePublicCreatesAFileInsideTheLastSharedPublicLinkFolderWithPass
 		);
 	}
 
+	/**
+	 * @When the public creates a file :file inside the last shared public link folder with password :password using wopi endpoint
+	 * @When the public tries to create a file :file inside the last shared public link folder with password :password using wopi endpoint
+	 *
+	 * @param string $file
+	 * @param string $password
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function thePublicCreatesAFileInsideTheLastSharedPublicLinkFolderWithPasswordUsingWopiEndpoint(string $file, string $password): void {
+		$this->createFile($file, $password);
+	}
+
+	/**
+	 * @When the public creates a file :file inside folder :folder in the last shared public link space with password :password using wopi endpoint
+	 * @When the public tries to create a file :file inside folder :folder in the last shared public link space with password :password using wopi endpoint
+	 *
+	 * @param string $file
+	 * @param string $folder
+	 * @param string $password
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function thePublicCreatesAFileInsideFolderInTheLastSharedPublicLinkSpaceWithPasswordUsingWopiEndpoint(string $file, string $folder, string $password): void {
+		$this->createFile($file, $password, $folder);
+	}
```

Comment:
Will it be possible to merge these two steps?

Reply:
is seems possible

Reply:
Please, don't merge if we have to add option params in-between other params.

Reply:
It is possible to merge but it seems to be better if we separate these two steps.